<?xml version="1.0" encoding="utf-8"?>
<TestRecord xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Results>
    <UnitTestResult>
      <TestDate>2012-11-14T20:18:21</TestDate>
      <Passed>0</Passed>
      <Errors>5</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-11-14T20:18:39</TestDate>
      <Passed>2</Passed>
      <Errors>3</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-11-14T20:18:43</TestDate>
      <Passed>2</Passed>
      <Errors>3</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-11-14T20:19:23</TestDate>
      <Passed>2</Passed>
      <Errors>3</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-11-14T20:32:58</TestDate>
      <Passed>5</Passed>
      <Errors>0</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>Test successful

Execution time: 0.97ms</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-11-14T20:35:35</TestDate>
      <Passed>5</Passed>
      <Errors>0</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>Test successful

Execution time: 0.95ms</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-11-14T21:04:26</TestDate>
      <Passed>5</Passed>
      <Errors>0</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>Test successful

Execution time: 0.89ms</Message>
    </UnitTestResult>
  </Results>
  <Tests>
    <TestRecord Name="ADODotNetAndMySQLTests">
      <Results>
        <UnitTestResult>
          <TestDate>2012-11-14T20:18:21</TestDate>
          <Passed>0</Passed>
          <Errors>5</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-11-14T20:18:39</TestDate>
          <Passed>2</Passed>
          <Errors>3</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-11-14T20:18:43</TestDate>
          <Passed>2</Passed>
          <Errors>3</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-11-14T20:19:23</TestDate>
          <Passed>2</Passed>
          <Errors>3</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-11-14T20:32:58</TestDate>
          <Passed>5</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0.96ms</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-11-14T20:35:35</TestDate>
          <Passed>5</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0.95ms</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-11-14T21:04:26</TestDate>
          <Passed>5</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0.88ms</Message>
        </UnitTestResult>
      </Results>
      <Tests>
        <TestRecord Name="Test">
          <Results>
            <UnitTestResult>
              <TestDate>2012-11-14T20:18:21</TestDate>
              <Passed>0</Passed>
              <Errors>5</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-11-14T20:18:39</TestDate>
              <Passed>2</Passed>
              <Errors>3</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-11-14T20:18:43</TestDate>
              <Passed>2</Passed>
              <Errors>3</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-11-14T20:19:23</TestDate>
              <Passed>2</Passed>
              <Errors>3</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-11-14T20:32:58</TestDate>
              <Passed>5</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0.94ms</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-11-14T20:35:35</TestDate>
              <Passed>5</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0.92ms</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-11-14T21:04:26</TestDate>
              <Passed>5</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0.86ms</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="TestCanCount">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:21</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>MySql.Data.MySqlClient.MySqlException : Unable to connect to any of the specified MySQL hosts.</Message>
                  <StackTrace>  at MySql.Data.MySqlClient.NativeDriver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Create (MySql.Data.MySqlClient.MySqlConnectionStringBuilder settings) [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.MySqlConnection.Open () [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:39</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.31ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:43</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.28ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:19:23</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.29ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:32:58</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.28ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:35:35</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.30ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T21:04:26</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.27ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="TestCanDelete">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:21</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>MySql.Data.MySqlClient.MySqlException : Unable to connect to any of the specified MySQL hosts.</Message>
                  <StackTrace>  at MySql.Data.MySqlClient.NativeDriver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Create (MySql.Data.MySqlClient.MySqlConnectionStringBuilder settings) [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.MySqlConnection.Open () [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:39</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanDelete () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:65 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:43</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanDelete () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:65 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:57</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanDelete () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:65 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:19:23</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanDelete () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:65 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:19:53</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanDelete () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:65 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:20:26</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Connection refused</Message>
                  <StackTrace>  at Mono.Remoting.Channels.Unix.HostConnectionPool.CreateConnection () [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Remoting.Channels.Unix.HostConnectionPool.GetConnection () [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Remoting.Channels.Unix.UnixConnectionPool.GetConnection (System.String path) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Remoting.Channels.Unix.UnixClientTransportSink.ProcessMessage (IMessage msg, ITransportHeaders requestHeaders, System.IO.Stream requestStream, ITransportHeaders&amp; responseHeaders, System.IO.Stream&amp; responseStream) [0x00000] in &lt;filename unknown&gt;:0 
  at System.Runtime.Remoting.Channels.BinaryClientFormatterSink.SyncProcessMessage (IMessage msg) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:32:58</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.19ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:35:35</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.16ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T21:04:26</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.15ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="TestCanDeleteAll">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:21</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>MySql.Data.MySqlClient.MySqlException : Unable to connect to any of the specified MySQL hosts.</Message>
                  <StackTrace>  at MySql.Data.MySqlClient.NativeDriver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Create (MySql.Data.MySqlClient.MySqlConnectionStringBuilder settings) [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.MySqlConnection.Open () [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:39</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.07ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:43</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.07ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:19:23</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.08ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:32:58</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.10ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:35:35</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.10ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T21:04:26</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.09ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="TestCanInsert">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:21</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>MySql.Data.MySqlClient.MySqlException : Unable to connect to any of the specified MySQL hosts.</Message>
                  <StackTrace>  at MySql.Data.MySqlClient.NativeDriver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Create (MySql.Data.MySqlClient.MySqlConnectionStringBuilder settings) [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.MySqlConnection.Open () [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:39</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanInsert () [0x00013] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:104 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:43</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanInsert () [0x00013] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:104 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:19:23</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanInsert () [0x00013] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:104 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:32:58</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.05ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:35:35</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.06ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T21:04:26</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.05ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="TestCanUpdate">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:21</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>MySql.Data.MySqlClient.MySqlException : Unable to connect to any of the specified MySQL hosts.</Message>
                  <StackTrace>  at MySql.Data.MySqlClient.NativeDriver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Open () [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.Driver.Create (MySql.Data.MySqlClient.MySqlConnectionStringBuilder settings) [0x00000] in &lt;filename unknown&gt;:0 
  at MySql.Data.MySqlClient.MySqlConnection.Open () [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:39</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanUpdate () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:122 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:18:43</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanUpdate () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:122 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:19:23</TestDate>
                  <Passed>0</Passed>
                  <Errors>1</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>System.IndexOutOfRangeException : Cannot find table -1</Message>
                  <StackTrace>  at System.Data.DataTableCollection.get_Item (Int32 index) [0x00000] in &lt;filename unknown&gt;:0 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryMySQLBase.GetDataTable (System.Data.DataSet dataSet, System.String tableName) [0x0000d] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryMySQLBase.cs:67 
  at ADODotNetAndMySQL.DAL.Repositories.RepositoryBase`1[ADODotNetAndMySQL.DAL.Models.SampleTable].GetAll () [0x0001f] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQL/DAL/Repositories/RepositoryBase.cs:58 
  at ADODotNetAndMySQLTests.Test.TestCanUpdate () [0x00011] in /PersonalData/DevProjects/SandBox/Git/DOT-NET-on-Linux/Day 8 ADO.NET &amp; MySQL/ADO.Net&amp;MySQL/ADO.Net&amp;MySQLTests/Test.cs:122 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:32:58</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.10ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T20:35:35</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.10ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-11-14T21:04:26</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.08ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
  </Tests>
</TestRecord>